### ISSUE TITLE
Semi-supervised finetuning on the LUNA dataset?

### ISSUE THREAD

**Issue by Luffy03** [RL4M/PCRLv2#7]
Thanks for your nice work!
How can we conduct semi-supervised finetuning on the LUNA dataset?

**Comment by liaochuanlin** [comment]
@Luffy03 Have you achieved the effect of the code?

**Comment by Luffy03** [comment]
> @Luffy03 Have you achieved the effect of the code?

No. Maybe something is wrong with my implementation. I have given up.

**Comment by liaochuanlin** [comment]
How much is the result of your realization about the brain?
&nbsp;


爩
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Jinheng ***@***.***&gt;; 
发送时间: 2023年9月27日(星期三) 中午12:02
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [RL4M/PCRLv2] Semi-supervised finetuning on the LUNA dataset? (Issue #7)





  
@Luffy03 Have you achieved the effect of the code?
  
No. Maybe something is wrong with my implementation. I have given up.
